packages/render-fidelity-tools/src/components/rendering-scenario.ts

```diff
@@ -39,27 +41,67 @@ export class RenderingScenario extends LitElement {
     return `./results/${this.name}`;
   }
 
+  updated(changedProperties: Map<any, any>) {
+    super.updated(changedProperties);
+
+    if (this.analysis == null) {
+      this.loadAnalysis();
+    }
+  }
+
+  private toDecibels(rmsDistanceRatio: number) {
+    return (10 * Math.log10(rmsDistanceRatio)).toFixed(2)
+  }
+
+  private async loadAnalysis() {
+    const analysisPath = `${this.basePath}/analysis.json`;
+    this.analysis = await (await fetch(analysisPath)).json();
+  }
+
   render() {
     const {basePath} = this;
     const {width} = this.dimensions;
 
-    const images = [{
-                     name: 'model-viewer',
-                     description: '<model-viewer> (this version)',
-                     file: 'model-viewer.png'
-                   }].concat(this.goldens)
-                       .filter(golden => !this.exclude.includes(golden.name))
-                       .map(golden => html`
+    const images =
+        [{
+          name: 'model-viewer',
+          description: '<model-viewer> (this version)',
+          file: 'model-viewer.png'
+        }].concat(this.goldens)
+            .filter(golden => !this.exclude.includes(golden.name))
+            .map(
+                (golden, index) => html`
 <div class="screenshot">
   <header>
     <h2>${golden.description}</h2>
   </header>
   <div class="check"></div>
   <img data-id="${this.name} ${golden.name}"
        style="width:${width}px" src="${basePath}/${golden.file}">
+    <div class = "metrics">
+      ${
+                    index == 0 || this.analysis == null ?
+                        html` <span> --- </span>` :
+                        html` <span>${
+                            this.toDecibels(
+                                this.analysis.analysisResults[index - 1][0]
+                                    .rmsDistanceRatio)} db
+             </span>
+              <div class="tooltip">
+                <span class="question-icon"> </span>
+                <span class="tooltiptext">
+                  Root mean square color distance between ${
```

Comment:
I would show the acronym: "Root mean square (RMS) Lab color distance", so that you can use that acronym below to shorten your equation: "10 * log(RMS)".